regression/library/test_manycore_vcache_sequence.c

```diff
@@ -70,22 +70,22 @@ int test_manycore_vcache_sequence() {
                 bsg_pr_test_info("%s: Testing DRAM bank (%" PRIu32 ",%" PRIu32 ")\n",
                                  __func__, dram_coord_x, dram_coord_y);
                 
-                for (size_t i = 0; i < ARRAY_LEN; i++) {
+                for (size_t i = 0; i < ARRAY_LEN; i += 1) {
                         if ((i % 64) == 1)
                                 bsg_pr_test_info("%s: Have written and read back %4zu words\n",
                                                  __func__, i);
                         
                         hb_mc_npa_t npa = hb_mc_npa_from_x_y(dram_coord_x,
                                                              dram_coord_y,
-                                                             BASE_ADDR+i);
+                                                             BASE_ADDR+(i << 2));
                         write_data = rand();
                         err = hb_mc_manycore_write_mem(mc, &npa, &write_data, sizeof(write_data));
                         if (err != HB_MC_SUCCESS) {
                                 bsg_pr_err("\n%s: failed to write A[%d] = 0x%08" PRIx32 ""
                                            " to DRAM coord (%d,%d) @ 0x08%" PRIx32 ": %s\n",
                                            __func__, i, write_data,
                                            dram_coord_x, dram_coord_y,
-                                           (npa.epa+i) << 2,
+                                           (npa.epa+ (i << 2)) << 2,
```

Comment:
I'm not sure why you left shift twice here... I think the EPA field stored as a byte address?

Reply:
Yes, you're right. i needs to be shifted (because it's a word index, not a byte address)

Reply:
Yes, I understanding shifting i, but I don't understand shifting the entire resulting address.

Reply:
I'm just writing poor responses right now...

Yes, I agree with you. I'll fix this.